.github/workflows/pr-stable-build.yml

```diff
@@ -0,0 +1,33 @@
+# allowing test inside PRs
+
+name: "PR build release"
+on:
+    push:
+        branches:
+          - '**'        # matches every branch
+          - '!master'   # excludes master
+
+env:
+    CI: false
+jobs:
+    publish-tauri:
```

Comment:
Misleading job name, since we don't actually publish. We just build.

```suggestion
    build-tauri:
```

Reply:
technically we're building it and publishing it to the releases no ?